rust/crd/src/lib.rs

```diff
@@ -130,11 +130,11 @@ impl SparkConfig {
         SparkConfigFragment {
             resources: ResourcesFragment {
                 cpu: CpuLimitsFragment {
-                    min: Some(Quantity("500m".to_owned())),
-                    max: Some(Quantity("1".to_owned())),
+                    min: Some(Quantity("100m".to_owned())),
```

Comment:
Can you please add a comment above similar to `This resources are used for the spark-submit Job`?